tests/model/test_model.py

```diff
@@ -3346,6 +3347,46 @@ def test__handle_subscription_event_subscribers_one_user_multiple_streams(
             new_subscribers = model.stream_dict[stream_id]["subscribers"]
             assert new_subscribers == expected_subscribers
 
+    @pytest.mark.parametrize(
+        "person, changed_details",
+        [
+            (
+                {"user_id": 11, "full_name": "New Full Name"},
```

Comment:
The user_id is common in all cases, so let's specify and build it in the test function.

Rather than setting the variable in the test function, you can alternatively list it as a default argument value, as long as it's a non-mutable value.